calculator.js

```diff
@@ -0,0 +1,19 @@
+function add (a, b) {
+    return a + b;
+}
+
+function sub(a, b) {
+    return a - b;
+}
+
+function multiplication(a,b) {
+    return a * b;
+}
+
+function division (a,b) {
+    return a / b;
+}
+
+function intDivision (a,b) {
+    return a % b;
```

Comment:
In this case the result of integer division should be 1 because integer division doesn't round the number.